serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -542,6 +542,10 @@ class MatrixSession(val client: HttpClient, val server: String, val user: String
         client.close()
         sync_should_run = false
     }
+    fun getRoomMembers(id: String): List<String> {
```

Comment:
As we mentioned, this should be pushed down so it can be shared with room names pr